lib/rucio/core/rule.py

```diff
@@ -3260,47 +3278,62 @@ def __create_recipents_list(rse_expression, filter_=None, *, session: "Session")
                 try:
                     email = get_account(account=account, session=session).email
                     if email:
-                        recipents.append((email, account))
+                        recipients.append((email, account))
                 except Exception:
                     pass
 
+    model = models.AccountAttrAssociation
```

Comment:
I have my reservations about this. Is your motivation to reduce the line length? It comes at the cost of readability.